sci-biology/nilearn/nilearn-0.2.4.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 python3_4 )
+
+inherit distutils-r1 git-r3
+
+DESCRIPTION="Fast and easy statistical learning on NeuroImaging data"
+HOMEPAGE="http://nilearn.github.io/"
+SRC_URI=""
+EGIT_REPO_URI="git://github.com/nilearn/nilearn"
+EGIT_COMMIT="${PV}"
```

Comment:
How is it live? It's identified via a specific commit, making it as dead as any other source - no? In any case, where would the snapshots be? do you mean someting like https://github.com/stnava/ANTs/archive/v2.1.0.tar.gz ?